cookbook/sonoff-dual-light-switch.rst

```diff
@@ -0,0 +1,270 @@
+DIY Light switch using a Sonoff Dual
+====================================
+
+.. seo::
+    :description: An example of how to integrate a dual light switch into Home Assistant using ESPHome
+    :image: sonoff_light_switch.png
+    :keywords: Relay, Sonoff Dual Dual R1, Light, HASS, Home Assistant, ESPHome
+
+.. note::
+
+    This is a DIY solution, and you will need to have some knowledge of electrical wiring and enough
+    capabilities to do this work safely.
+
+    The author, and the ESPHome team, take no responsibility for any actions, injuries or outcomes
+    from following this guide.
+
+    In some countries you may need specific qualifications before you can carry out such work in
+    a residentaial property.
+
+Please read up on :doc:`/cookbook/sonoff-basic-light-switch` to get the background and principals of
+the this project. Its all basically the same, but with a double switch.
+
+So we will be using GPIO4 and GPIO14 for the two retractive switches, again they will both short to 0v
+when the switch is clicked.
+
+R1
+--
+
+The R1 version of the Dual controls the relays via the UART, so the code gets a bit complex here.
+
+.. code-block:: yaml
+
+    esphome:
+      name: dual_ls
+      platform: ESP8266
+      board: esp01_1m
+
+    wifi:
+      ssid: !secret wifi_ssid
+      password: !secret wifi_password
+      fast_connect: True
```

Comment:
Is there a reason for these fast connects? I would prefer not to have them here, as fast_connect disables some functionality like round-robin wifi network selection that makes wifi more stable.

Reply:
Its because my HASS wifi is hidden, so its required, and I just copied my files. Clearly not needed for an example, and I shall remove them